docs/README.md

```diff
@@ -1,29 +1,22 @@
 # Unofficial Talon Docs
 
-Documenting my learning experiences with Talon, the speech-to-text system. These docs are not authorative or complete, but instead represent an 80/20 view of the API sufficient for getting started making extensions and using the system.
+Documenting my learning experiences with Talon, the speech-to-text system. These docs are not authorative or complete. After reading them you should be able to extend and use Talon.
 
 
 ## Intro
 
-Talon is a next-generation speech-to-text system for coding, commanding, and dictating to a computer. It is currently in open beta and is under active development by creator, Ryan Hileman, as of April 2018.
+Talon is a speech-to-text system for commanding and dictating to a computer, aimed at helping programmers write code without a keyboard. It is currently in open beta and is under active development by creator Ryan Hileman as of March 2019.
```

Comment:
I think this is an unnecessary reduction in the potential reach of Talon. I see no reason why it should be considered only usable by programmers.